src/config/mockQuizData.ts

```diff
@@ -0,0 +1,135 @@
+export const mockQuizData = {
+  questions: [
+    {
+      id: 1,
+      question: "What is the capital of France?",
+      type: "multiple",
+      options: ["Berlin", "Madrid", "Paris", "Rome"],
+      correctAnswer: "Paris"
+    },
+    {
+      id: 2,
+      question: "What is the capital of Germany?",
+      type: "multiple",
+      options: ["Berlin", "Vienna", "Bern", "Brussels"],
+      correctAnswer: "Berlin"
+    },
+    {
+      id: 3,
+      question: "What is the capital of Spain?",
+      type: "multiple",
+      options: ["Lisbon", "Madrid", "Barcelona", "Valencia"],
+      correctAnswer: "Madrid"
+    },
```

Comment:
how about refactiring here since you have duplications in options?
sort of
```ts
interface ContentType {
  id: number;
  content: string;
}
const QUESTIONS: ContentType[] = [
  { id: 1, content: "What is the capital of France?" },
  { id: 2, content: "What is the capital of Germany?" },
  //... other questions
];

const ANSWERS: ContentType[] = {
  1: 'Berlin',
  2: 'Madrid',
  3: 'Paris',
  4: 'Valencia',
}

const QUESTION_ANSWERS_MAP = {
  1:  [1, 2, 3, 4],
  2:  [1, 2, 3, 4],
}

const CORRECT_ANSWERS = [
  { questionId: 1, correctAnswerId: 3 }
];

const getAnswersByQuesstionId = (questionId: number) => {
  const questionAnswerItem = QUESTION_ANSWERS_MAP[questionId];
  return questionAnswerItem?.map(id => ANSWERS[id]);
}
const checkIfAnswerIsCorrect = (questionId: number, answerId: number) => {
  const correctAnswer = CORRECT_ANSWERS.find(item => item.questionId === questionId);
  return correctAnswer?.correctAnswerId === answerId;
}

```

Reply:
Is it really necessary to refactor mockQuizData? There are only three duplications in the options. And such a refactoring will make the information for the correct answers unreadable, from my point of view =)